cmd/run.go

```diff
@@ -76,8 +76,9 @@ func init() {
 	flagSet.String("geth.ipcpath", "", "Path to the geth ipc file")
 	utils.ExitOnError(serveCmd.MarkFlagFilename("geth.ipcpath"))
 
-	flagSet.String("geth.genesis", "", "path to the genesis.json")
+	flagSet.String("geth.genesis", "", "(Optional) path to the genesis.json, for use with custom chains")
 	utils.ExitOnError(serveCmd.MarkFlagFilename("geth.genesis", "json"))
+	flagSet.String("geth.network", "", "Network to use, either 'mainnet', 'alfajores', or 'baklava'")
```

Comment:
This should also be marked a optional, since you specify either this or geth.genesis.